vhdbuilder/packer/windows/components-test.json

```diff
@@ -1,5 +1,48 @@
 {
   "ContainerImages": [
+    {
+      "downloadURL": "mcr.microsoft.com/container/with/seperate/win/and/linux/versions:*",
```

Comment:
Q: what are this test file for? The downloadURL is a little weird to me.

Reply:
The test file is just to check the code that can parse and process components.json works properly. The URL is designed to look like a test URL so devs don't feel the need to update it.

Reply:
@timmy-wright could you move them to vhdbuilder/packer/test since they are test files? I think that it should not confuse anyone if they are in test folder.